OracleTuxedo/art_runtime/dockerfiles/12.2.2/Dockerfile.template

```diff
@@ -0,0 +1,84 @@
+#
+# Dockerfile template for Tuxedo ART 12.2.2
+# 
+# For Tuxedo 12.2.2, you need JRE installed first, so before you can build this image, you must download the Oracle Server JRE binary and drop in folder OracleJDK/java-8 and build that image.
```

Comment:
This image extends the `oracle/tuxedo` image, which already has the Server JRE installed. This entire comment is not necessary, neither are lines 5 and 6.